netlify.toml

```diff
@@ -11,3 +11,17 @@ HUGO_VERSION = "0.69.0"
 
 [context.branch-deploy.environment]
 HUGO_VERSION = "0.69.0"
+
+[[redirects]]
+from = "/nn"
+to = "https://forms.nycmesh.net/nn-assign"
+status = 302
+force = true
+#redirect requests for nycmesh.net/nn to MeshDB NN assignment page
+
+[[redirects]]
+from = "/query"
+to = "https://forms.nycmesh.net/query"
+status = 302
+force = true
+#redirect requests for nycmesh.net/query to MeshDB NN assignment
```

Comment:
Can we do these in `static/_redirects` instead?

Reply:
Done